server/svix-server/config.default.toml

```diff
@@ -5,6 +5,12 @@
 # The address to listen on
 listen_address = "0.0.0.0:8071"
 
+# The address to send operational webhooks to. When not present, operational webhooks will not be
+# sent. When present, the API server with the given URL will be used to send operational webhooks.
+# This should be the address of the Svix API server that this application is running. It is
+# configurable for the case of a worker running independently of an API server.
```

Comment:
```suggestion
# The address of the Svix server to use for sending operational webhooks (disabled when omitted/null)
# Operational webhooks (otherwise known as "incoming webhooks"), are webhooks
# send from the Svix server to you, to let you know when some events happen.
# For a list of supported events please refer to: https://api.svix.com/docs#tag/Webhooks
```